core/arch/arm/plat-bcm/crc32.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2019 Broadcom.
+ */
+
+#ifndef CRC32_H
+#define CRC32_H
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#define CRC32_INIT_VAL	(~0)
+#define CRC32		crc32i
+
+uint32_t ucrc32(const char ch, uint32_t crc);
```

Comment:
`ucrc32()`  not to be exported. Make it static in  C file and remove from here.

Reply:
Done. Made ucrc32 static function.